lib/yang_wrappers.h

```diff
@@ -114,6 +114,8 @@ extern const char *yang_get_default_string(const char *xpath_fmt, ...);
 extern void yang_get_default_string_buf(char *buf, size_t size,
 					const char *xpath_fmt, ...);
 
+extern struct yang_data *yang_data_new_empty(const char *xpath);
```

Comment:
nit: please add a `/* empty */` comment before this line.

Reply:
added.